src/commands/registerCommands.ts

```diff
@@ -328,6 +329,60 @@ function registerAssociationCommands(context: ExtensionContext, languageClient:
 
 }
 
+interface ConfigurationItemEdit {
+  section: string;
+  value: any;
+  editType: ConfigurationItemEditType;
+}
+
+enum ConfigurationItemEditType {
+  Add = 0,
+  Delete = 1
+}
+
+/**
+ * Register commands used for registering/unregistering catalog for command menu and CodeLens
```

Comment:
It is not only for catalog, it is a command to update configuration on client side.